pom.xml

```diff
@@ -423,6 +423,12 @@ limitations under the License.
       <version>${koverse.version}</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-mllib_2.10</artifactId>
+      <version>1.6.1</version>
```

Comment:
I think we want this marked provided as it will be available at runtime on the cluster.